src/pages/LenderProfile/LenderProfile.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+	<www-page>
+		<kv-page-container>
+			<!-- Page Content -->
+		</kv-page-container>
+	</www-page>
+</template>
+
+<script>
+import WwwPage from '@/components/WwwFrame/WwwPage';
+import lenderPublicProfileQuery from '@/graphql/query/lenderPublicProfile.graphql';
+import KvPageContainer from '~/@kiva/kv-components/vue/KvPageContainer';
+
+export default {
+	name: 'LenderProfile',
+	inject: ['apollo', 'cookieStore'],
+	components: {
+		WwwPage,
+		KvPageContainer,
+	},
+	metaInfo() {
+		return {
+			title: this.pageTitle,
+			meta: [
+				{ property: 'og:title', vmid: 'og:title', content: this.pageTitle },
+				{ property: 'og:description', vmid: 'og:description', content: this.pageDescription },
+				{ property: 'og:site_name', vmid: 'og:site_name', content: 'Kiva' },
+				{
+					vmid: 'description',
+					name: 'description',
+					content: this.pageDescription,
+				}
+			].concat([
+				{
+					vmid: 'facebook_label',
+					name: 'facebook_label',
+					content: `Kiva - Lender > ${this.lenderName} from ${this.lenderWhereAbouts}`
+				},
+			]).concat([
+				{
+					vmid: 'robots',
+					name: 'robots',
+					content: 'noindex, nofollow',
+				},
+			]),
+		};
+	},
+	data() {
+		return {
+			lenderInfo: {},
+		};
+	},
+	apollo: {
+		preFetch(config, client, { route }) {
```

Comment:
Do we want to prefetch? Theoretically we could cache the lender profiles, since most of the data doesn't change too often.